plugins/CMakeLists.txt

```diff
@@ -283,6 +283,7 @@ REGISTER_IN_PLUGIN("in_random")
 # PROCESSORS
 # ==========
 REGISTER_PROCESSOR_PLUGIN("processor_content_modifier")
+REGISTER_PROCESSOR_PLUGIN("processor_calyptia")
```

Comment:
Surrunding with if(FLB_LUAJIT) would be nice instead of adding filter part on this file.
```
if(FLB_LUAJIT)
  REGISTER_PROCESSOR_PLUGIN("processor_calyptia")
endif()
```